channels/chan_iax2.c

```diff
@@ -1447,6 +1447,7 @@ static int iax2_is_control_frame_allowed(int subtype)
 	case AST_CONTROL_STREAM_RESTART:
 	case AST_CONTROL_STREAM_REVERSE:
 	case AST_CONTROL_STREAM_FORWARD:
+	case AST_CONTROL_PLAYBACK_BEGIN:
```

Comment:
It should be documented that AST_CONTROL_PLAYBACK_BEGIN is only supported by app_dial.